src/analyses/libraryFunctions.ml

```diff
@@ -508,6 +566,7 @@ let linux_userspace_descs_list: (string * LibraryDesc.t) list = LibraryDsl.[
     ("fts_open", unknown [drop "path_argv" [r_deep]; drop "options" []; drop "compar" [s]]); (* TODO: use Call instead of Spawn *)
     ("fts_read", unknown [drop "ftsp" [r_deep; w_deep]]);
     ("fts_close", unknown [drop "ftsp" [f_deep]]);
+    ("getdtablesize", unknown []);
```

Comment:
This doesn't seem specific to the Linux kernel's userspace, but maybe just glibc?